cmd/sysl/cmd_databasescript.go

```diff
@@ -0,0 +1,103 @@
+package main
```

Comment:
Lets not add everything to the main package, the cmd could reside in the main package but the core functionality should be in a app specific package.

Reply:
There is .DS_Store file accidentally checked in, please remove that